src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
```

Comment:
Since 'vocationCost' is only assigned once and not reassigned, it should be declared as 'const' instead of 'let'.